runtime/acala/src/lib.rs

```diff
@@ -728,6 +729,31 @@ impl DataFeeder<CurrencyId, Price, AccountId> for AggregatedDataProvider {
 	}
 }
 
+pub struct ExistentialDepositsForDropAssets;
+impl ExistentialDepositsForDropAssets {
+	fn get(currency_id: &CurrencyId) -> Balance {
+		match currency_id {
+			CurrencyId::Token(symbol) => match symbol {
+				TokenSymbol::ACA => NativeTokenExistentialDeposit::get(),
```

Comment:
can we reuse the values instead of duplicate them?
i.e. only handle ACA and have default arm send to the other ExistentialDeposits

Reply:
yeah, doing it now.